tests/MongoFramework.Benchmarks/MongoFramework.Benchmarks.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
You can't set an output application as .NET Standard - only libraries etc support it. An application needs a specific target - probably best to set this to .NET 6 now.